mysql-test/suite/rocksdb/t/rocksdb_checksums.test

```diff
@@ -121,8 +153,8 @@ select a from t3 force index(a) where a<4;
 select a from t4 force index(a) where a<1000000;
 set session debug= "-d,myrocks_simulate_bad_key_checksum1";
 
-set @@global.rocksdb_store_row_debug_checksums=@save_rocksdb_store_row_debug_checksums;
-set @@global.rocksdb_verify_row_debug_checksums=@save_rocksdb_verify_row_debug_checksums;
-set @@global.rocksdb_checksums_pct=@save_rocksdb_checksums_pct;
-
+#cleanup
```

Comment:
Nope, it is not...good catch...--repeat missed it because the test clears it before each search...still sloppy though and needs cleanup.